flask_core/middleware/handler.py

```diff
@@ -9,11 +9,12 @@ class Handler(object):
     def __init__(self, wsgi_app):
         self.wsgi_app = wsgi_app
         self.middleware = [
-            FilterMiddleware(self.wsgi_app),
-            AuthMiddleware(self.wsgi_app),
-            IsolationMiddleware(self.wsgi_app),
+            FilterMiddleware(self.wsgi_app)
         ]
-
+        if not bool(os.environ.get("DISABLE_AUTH", False)):
```

Comment:
Set this in config.py please. It can default to auth disabled.

You can access the `app` object through `wsgi_app.__self__`.